workflows/eager.nf

```diff
@@ -123,6 +124,14 @@ workflow EAGER {
         if ( params.preprocessing_tool == 'fastp' && !adapterlist.extension.matches(".*(fa|fasta|fna|fas)") ) error "[nf-core/eager] ERROR: fastp adapter list requires a `.fasta` format and extension (or fa, fas, fna). Check input: --preprocessing_adapterlist ${params.preprocessing_adapterlist}"
     }
 
+    // Contamination estimation
+    if ( params.run_contamination_angsd ) {
+        hapmap_file = file(params.angsd_hapmap, checkIfExists:true)
+        ch_hapmap = Channel.of( [ hapmap_file ] )
+    } else {
+        ch_hapmap = Channel.empty()
+    }
+
```

Comment:
Does it even need to be read into a channel actually? 🤔  I think if you combine the hapmap ONTO the BWA indexes, instead of the other way around, it won't complain about method invocation.